docs/_data/portland-2024-config.yaml

```diff
@@ -43,12 +43,12 @@ buttons:
     # - text: Conference recap
     #   link: /news/thanks-recap
   bottom:
-    # - text: Welcome to our conference
-    #   link: /news/welcome
+    - text: Welcome to our conference
+      link: /news/welcome
```

Comment:
I think we probably just want a ticket link here too, and not a link to our initial news post.

Reply:
Pushed up a fix 👍

Reply:
Sounds good - merge when you're fully futzed?